en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/as-a-standalone-gateway.md

```diff
@@ -16,17 +16,17 @@ In this case that would be the OpenAPI definition of backend service or microser
 
 - Different types of endpoints
 - Gateway security
-- Rate limiting...
+- Rate limiting
```

Comment:
There are some similar changes here which has been merged to 4.0.0 https://github.com/wso2/docs-apim/pull/5090

Reply:
+1. Noticed while creating the PR for 4.0.0 branch.